app/mailers/user_mailer.rb

```diff
@@ -9,6 +9,19 @@ def email_changed(old_email)
     mail(to: old_email, subject: t('mailer.email_change_notice.subject'))
   end
 
+  # :reek:ControlParameter
+  # :reek:LongParameterList
+  # :reek:TooManyStatements
```

Comment:
I assume you want to leave in the 3 lines above. I am pointing this out in case you intended to remove them later.

Reply:
Yes, I meant to disable these. Normally I'd add a method for consuming the parameters, but this class is already pretty fat and I'd be worried about that getting lost in the concerns it has for the other mails it sends.